publication-commons/src/test/resources/features/CreatorAccessRights.feature

```diff
@@ -0,0 +1,148 @@
+    When UPDATE is called to update the publication according to the new content on behalf of the authenticated user
+    Then the existing publication is updated according to the new content
+    And the new version of the publication replaces the old version in the database
+    And UPDATE returns the old version of the publication
+
+    Examples:
+      | status    |
+      | DRAFT     |
+      | PUBLISHED |
+
+  Scenario: users with CREATOR role publish own draft publication
+    Given an existing publication
+    And the status of the existing publication is DRAFT
+    And the owner of the publication is the authenticated user
+    And there is an update on the publication's content
+    When PUBLISH is called to publish the publication on behalf of the authenticated user
+    Then the existing publication status is set to PUBLISH
+    And the status is updated in the database
+    And PUBLISH returns the published version of the publication
+
+  Scenario: users with CREATOR role deletes own draft publication
+    Given an existing publication
+    And the status of the existing publication is DRAFT
+    And the owner of the publication is the authenticated user
+    When DELETE is called to delete the publication on behalf of the authenticated user
+    Then the publication is deleted from the database
+    And DELETE returns the last saved version of the publication
+
+  Scenario: users with CREATOR role cannot delete own published publication
+    Given an existing publication
+    And the status of the existing publication is PUBLISHED
+    And the owner of the publication is the authenticated user
+    When DELETE is called to delete the publication on behalf of the authenticated user
+    Then DELETE returns an error indicating that the user is not allowed to perform this action
+
+
+  Scenario: users with role USER can see published material when they list publications
+    Given that DynamoDBPublicationService has a LIST method
+    And that LIST requires a user with non empty username
+    And that LIST requires a publication owner's username
+    And a user with username "theCreator" that owns some publications
+    When LIST is called to list the publication of the user "theCreator" on behalf of the authenticated user
+    Then LIST returns all published publications whose owner is the user "theCreator"
+
+
+#====================================PROHIBITIONS=================================================
+
+  Scenario: users with CREATOR role cannot read not owned draft publication
+    Given an existing publication
+    And the status of the existing publication is DRAFT
+    And the owner of the publication is not the authenticated user
+    When READ is called to read the publication on behalf of the the authenticated user
+    Then READ returns that the publication was not found.
+
+
+  Scenario Outline: forbidden non read actions on NOT owned draft publications for users with CREATOR role
```

Comment:
User with CreatorRole attempts to perform action on other Users' unshared Draft Publications